src/gamma.jl

```diff
@@ -720,6 +742,31 @@ Euler integral of the first kind ``\\operatorname{B}(x,y) = \\Gamma(x)\\Gamma(y)
 function beta end
 
 function beta(x::Real, w::Real)
```

Comment:
lets change the arguments to `a,b` to be consistent with the incomplete beta ones.